frontend/.eslintrc.json

```diff
@@ -1,50 +1,46 @@
 {
-    "root": true,
-    "ignorePatterns": [
-      "projects/**/*"
-    ],
-    "overrides": [
-      {
-        "files": [
-          "*.ts"
+  "root": true,
+  "ignorePatterns": [
+    "projects/**/*"
+  ],
+  "overrides": [
+    {
+      "files": [
+        "*.ts"
+      ],
+      "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended",
+        "plugin:@angular-eslint/recommended",
+        "plugin:@angular-eslint/template/process-inline-templates"
+      ],
+      "rules": {
+        "@angular-eslint/directive-selector": [
+          "error",
+          {
+            "type": "attribute",
+            "prefix": "dh",
+            "style": "camelCase"
+          }
         ],
-        "parserOptions": {
-          "project": [
-            "tsconfig.json"
-          ],
-          "createDefaultProgram": true
-        },
-        "extends": [
-          "plugin:@angular-eslint/recommended",
-          "plugin:@angular-eslint/template/process-inline-templates"
-        ],
-        "rules": {
-          "@angular-eslint/directive-selector": [
-            "error",
-            {
-              "type": "attribute",
-              "prefix": "dh",
-              "style": "camelCase"
-            }
-          ],
-          "@angular-eslint/component-selector": [
-            "error",
-            {
-              "type": "element",
-              "prefix": "dh",
-              "style": "kebab-case"
-            }
-          ]
-        }
-      },
-      {
-        "files": [
-          "*.html"
-        ],
-        "extends": [
-          "plugin:@angular-eslint/template/recommended"
-        ],
-        "rules": {}
+        "@angular-eslint/component-selector": [
+          "error",
+          {
+            "type": "element",
+            "prefix": "dh",
+            "style": "kebab-case"
+          }
+        ]
       }
-    ]
-  }
+    },
+    {
+      "files": [
+        "*.html"
+      ],
+      "extends": [
+        "plugin:@angular-eslint/template/recommended"
+      ],
+      "rules": {}
+    }
+  ]
+}
```

Comment:
I see the additions to the `extends` section here, which look good, but what are the other changes?

Reply:
I fixed an indenting error, which makes Git think that the whole file was changed. The only other change here is the removal of `parserOptions`. I did not do this deliberately, but I think it was done automatically by adding the `@typescript-eslint/recommended` plugin.